config/crd/bases/sriovnetwork.openshift.io_sriovibnetworks.yaml

```diff
@@ -1,6 +1,6 @@
 
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
I think this is not related to the switchdev

Reply:
There are several CRD files changes not related to eswitch mode, it seems you are not using the same version of the operator-sdk, IMO it can be in another PR

Reply:
Yeah, it can be in a separate PR. Those changes were mistakenly introduced by the previous operator upgrade.